packages/sdks/snippets/vue/src/views/LivePreview.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div v-if="loading">Loading Data...</div>
+  <div v-else-if="content" class="blog-data-preview">
+    <div>Blog Title: {{ content?.data?.title }}</div>
+    <div>Authored by: {{ content?.data?.author }}</div>
+    <div>Handle: {{ content?.data?.handle }}</div>
+    <div>Published date: {{ publishedDate }}</div>
+  </div>
+  <div v-else class="no-data-message">No Data.</div>
+</template>
+
+<script setup lang="ts">
+import { ref, computed, onMounted, onBeforeUnmount, watch } from 'vue'
+import { useRoute } from 'vue-router'
+import { fetchOneEntry, subscribeToEditor } from '@builder.io/sdk-vue'
+
+const content = ref<any>(null)
+const loading = ref(true)
+
+const route = useRoute()
+
+function getSlug() {
+  return route.path
+}
```

Comment:
can we remove this helper? it's only used once and adds noise. did you mean to make it a `computed()` and use it in `watch()` like in the nuxt snippet?